test/e2e/charts/clickhouse-tls/values.yaml

```diff
@@ -0,0 +1,23 @@
+# Settings for ingress
+ingress:
+  apiVersion: networking.k8s.io/v1
```

Comment:
since this is for testing (and not meant for users), I suggest removing all the values you don't actually need, such as `apiVersion`, etc

Reply:
I've removed all of them except ingress.enable. Thanks.